docs/docker-compose/README.md

```diff
@@ -21,6 +21,12 @@ As well as the prerequisites mentioned on the [main README](/README.md#prerequis
 * [Docker](https://www.docker.com/products/docker-desktop)
 * [Docker Compose](https://docs.docker.com/compose/install)
 
+## Configure Docker for Desktop
+
+In order to deploy onto Docker for Desktop we need to allocate at least [13 Gb](../../docker-compose/docker-compose.yml) (preferably 16 Gb) to the Docker Engine on the "Resources" tab in Docker for Desktop's preferences pane as shown in the screenshot below.
```

Comment:
Maybe add note to say that otherwise insufficient memory may cause containers to exit without warning.